templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -186,20 +197,33 @@
 						{{ctx.Locale.Tr "repo.issues.new.closed_projects"}}
 					</div>
 					{{range .ClosedProjects}}
-						<a class="item muted sidebar-item-link" data-id="{{.ID}}" data-href="{{.Link ctx}}">
+						{{$ProjectID := $.Projects.IssueID}}
+						{{$checked := false}}
+						{{range $.Issue.Projects}}
+							{{if eq .IssueID $ProjectID}}
+								{{$checked = true}}
+								{{break}}
+								{{end}}
```

Comment:
What's the point of this loop?

Reply:
Maybe, but I don't know how to implement it?

Reply:
https://github.com/go-gitea/gitea/blob/main/templates/repo/issue/view_content/sidebar.tmpl#L229C7-L232C12
The assigned fields also use the loop method

Reply:
Seems ok then. In JS i would just do [`arr.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) but of course golang or it's templates are not much into functional programming 😆.